### ISSUE TITLE
Permalinks on Evolution dashboard do not retain selected aggregates for categorical histogram

### ISSUE THREAD

**Issue by mreid-moz** [mozilla/telemetry-dashboard#395]
STR:
1. visit [this permalink](https://mzl.la/2EWOIGN)
2. un-select "eOK" key and instead select "eTooLate"
3. generate a new permalink
4. load the new permalink
5. Observe selected keys (it reverts to "eOK" for me).

**Comment by chutten** [comment]
Probably fallout from #377 where I reworked how we load and change measures in the evo dashboard.

**Comment by ab0092** [comment]
@chutten i have gone through the issue and i think the selected aggregate type is not part of the url which is used to restore state of the page. Is it something you are working on or shall i work on this issue

**Comment by o0Ignition0o** [comment]
It seems like the aggregates are part of the url, eg : "http://0.0.0.0:8000/new-pipeline/evo.html#!aggregates=0!4" but they don't get parsed the right way. 

I also don't get what's going on [here](https://github.com/mozilla/telemetry-dashboard/blob/gh-pages/new-pipeline/src/dashboards.js#L229), it seems like the aggregates key have nothing to do with the applicable values.

This is pretty weird indeed ! :)

I should probably also file a (maybe) followup bug because it seems like I have to click on the aggregate selector twice before it selects anything O_o

**Comment by chutten** [comment]
Oh geez, the aggregates are saved to the URL using their index? That's... probably "mostly fine" but certainly isn't parsed that way.

**Comment by gargshruti30** [comment]
@chutten @georgf Can I work on this issue ?

**Comment by chutten** [comment]
Yes you may. It might prove tricky. You may want to start by looking at #404 and at the code in `loadStateFromUrlAndCookie`
